core/src/main/java/org/apache/carbondata/scan/processor/AbstractDataBlockIterator.java

```diff
@@ -127,11 +133,15 @@ protected boolean updateScanner() {
     }
   }
 
-  private AbstractScannedResult getNextScannedResult() throws QueryExecutionException {
+  private AbstractScannedResult getNextScannedResult(QueryStatisticsRecorder recorder,
```

Comment:
Why we need to change this getNextScannedResult() method  parameters. if required please pass a statistics model, this will make sure that our method parameters wont grow